Test failure on 32bit

```
___________________________________ TestLdCalculator.test_get_r2_array ___________________________________
self = <tests.test_lowlevel.TestLdCalculator object at 0xaeae5ef0>
    def test_get_r2_array(self):
        ts = self.get_example_tree_sequence()
        calc = _tskit.LdCalculator(ts)
        with pytest.raises(TypeError):
            calc.get_r2_array()
        with pytest.raises(TypeError):
            calc.get_r2_array(None)
        # Doesn't support buffer protocol, so raises typeerror
        with pytest.raises(TypeError):
            calc.get_r2_array(None, 0)
        n = ts.get_num_sites()
        assert n > 2
        with pytest.raises(BufferError):
            calc.get_r2_array(bytes(100), 0)
        buff = bytearray(1024)
        with pytest.raises(ValueError):
            calc.get_r2_array(buff, 0, max_distance=-1)
        with pytest.raises(ValueError):
            calc.get_r2_array(buff, 0, direction=1000)
        # TODO this API is poor, we should explicitly catch these negative
        # size errors.
        for bad_max_mutations in [-2, -3, -(2 ** 32)]:
            with pytest.raises(BufferError):
>               calc.get_r2_array(buff, 0, max_mutations=bad_max_mutations)
E               OverflowError: Python int too large to convert to C ssize_t
tests/test_lowlevel.py:1759: OverflowError
======================================== short test summary info =========================================
FAILED tests/test_lowlevel.py::TestLdCalculator::test_get_r2_array - OverflowError: Python int too larg...
================== 1 failed, 2624 passed, 10 skipped, 1 xfailed in 55735.74s (15:28:55) 
```

This is really just a bad test case, but I'll fix and verify on my laptop.

--- a/python/tests/test_lowlevel.py
+++ b/python/tests/test_lowlevel.py
@@ -1752,9 +1752,10 @@ class TestLdCalculator(LowLevelTestCase):
             calc.get_r2_array(buff, 0, max_distance=-1)
         with pytest.raises(ValueError):
             calc.get_r2_array(buff, 0, direction=1000)
+
         # TODO this API is poor, we should explicitly catch these negative
         # size errors.
-        for bad_max_mutations in [-2, -3, -(2 ** 32)]:
+        for bad_max_mutations in [-2, -3]:
             with pytest.raises(BufferError):
                 calc.get_r2_array(buff, 0, max_mutations=bad_max_mutations)
         for bad_start_pos in [-1, n, n + 1]:

